app/helpers/teams_helper.rb

```diff
@@ -12,4 +12,32 @@ def role_within_team(team)
       "-"
     end
   end
+
+  # Render the namespace scope icon.
+  def team_scope_icon(team)
+    icon = "fa-user"
+    title = "Personal"
+
+    if team.team_users.count > 1
+      icon = "fa-users"
+      title = "Team"
+    end
+
+    content_tag :i, "", class: "fa #{icon} fa-lg", title: title
+  end
+
+  # Render the team user role icon.
+  def team_user_role_icon(team_user)
+    icon = "fa-times-circle"
```

Comment:
Would you prefer? The second seems weird to me, because variable scope seems implicit. 

```ruby
    icon = case team_user.role
           when "owner"
             "fa-bolt"
           when "contributor"
             "fa-exchange"
           when "viewer"
             "fa-eye"
           else
             "fa-times-circle"
           end
```
or
```ruby
    case team_user.role
    when "owner"
      icon = "fa-bolt"
    when "contributor"
      icon = "fa-exchange"
    when "viewer"
      icon = "fa-eye"
    else
      icon = "fa-times-circle"
    end
```